integration_tests/tests/integrity/zontal_integrity_daily_shop_orders.sql

```diff
@@ -0,0 +1,63 @@
+{{ config(
+    tags="fivetran_validations",
+    enabled=var('fivetran_validation_tests_enabled', false)
+) }}
+
+with orders as (
+
+    select *
+    from {{ ref('shopify__orders') }}
+),
+
+order_metrics as (
+
+    select 
+        source_relation,
+        cast({{ dbt.date_trunc('day','created_timestamp') }} as date) as date_day,
+        count(distinct order_id) as count_orders,
+        sum(order_adjusted_total) as order_adjusted_total
+
+    from orders
+    where created_timestamp > '2023-01-01' and created_timestamp < '2024-01-01'
+    group by 1,2
+),
+
+daily_shop as (
+
+    select *
+    from {{ ref('shopify__daily_shop') }}
+),
+
+daily_shop_metrics as (
+
+    select 
+        source_relation,
+        date_day,
+        sum(count_orders) as count_orders,
+        sum(order_adjusted_total) as order_adjusted_total
+
+    from daily_shop
+    where date_day >= '2023-01-01' and date_day < '2024-01-01'
+    group by 1,2
+),
+
+final as (
+
+    select
+        daily_shop_metrics.source_relation,
+        daily_shop_metrics.count_orders as daily_shop_count_orders,
+        order_metrics.count_orders as order_count_orders,
+        daily_shop_metrics.order_adjusted_total as daily_shop_order_adjusted_total,
+        order_metrics.order_adjusted_total as order_order_adjusted_total
```

Comment:
We should add a coalesce here because if there are no orders at the source then the above ctes will produce `null` records here. To ensure the where clause at the bottom is always accurate it would be best to treat a `null` record as a `0`.

![image](https://github.com/fivetran/dbt_shopify/assets/74217849/35ecff92-081c-4e39-b087-fcdbbf0ffad5)


```suggestion
        coalesce(order_metrics.count_orders, 0) as order_count_orders,
        daily_shop_metrics.order_adjusted_total as daily_shop_order_adjusted_total,
        coalesce(order_metrics.order_adjusted_total, 0) as order_order_adjusted_total
```

Reply:
added myself